apps/cyberstorm-remix/app/commonComponents/PackageSearch/PackageSearch.css

```
@@ -166,17 +135,33 @@
-      justify-content: center;
+  .package-search__pagination {
+    display: flex;
+    flex-direction: column;
+    gap: var(--gap-xl);
+    align-items: center;
+    justify-content: center;
+    padding: var(--space-48) 0;
+    color: var(--color-text-tertiary);
+    font-weight: var(--font-weight-regular);
+
+    font-size: var(--font-size-body-md);
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)